tests/build/mod.rs

```diff
@@ -1,168 +1,237 @@
+use oranda::site::layout::Layout;
+
 mod fixtures;
 use super::utils::tokio_utils::TEST_RUNTIME;
 use fixtures::{oranda_config, page};
 
+fn temp_build_dir() -> String {
+    assert_fs::TempDir::new()
+        .unwrap()
+        .to_str()
+        .unwrap()
+        .to_string()
+}
+
```

Comment:
```suggestion
fn temp_build_dir() -> (TempDir, String) {
    let dir = assert_fs::TempDir::new().unwrap();
    (dir, dir.to_str().unwrap().to_string())
}
```

Yeah this won't work. The dir gets cleaned up by TempDir's Drop. You need to return it.

Caller would then do like

```rs
let (_guard, temp_dir) = temp_build_dir();
```

Alternatively only return the TempDir and make the user do .to_str() themselves